core/client/tests/unit/components/gh-profile-image-test.js

```diff
@@ -28,7 +28,10 @@ describeComponent(
                 testEmail = 'test@example.com',
                 style, size;
 
-            component.set('email', testEmail);
+            Ember.run(function () {
+                component.set('email', testEmail);
+            });
+
```

Comment:
tried all kinds of approaches and ran out of idea about how to test this debounced property setter. this is the mose sane approach that should've worked, but it didn't. also tried mocking Ember.run.debounce and using `sinon.useFakeTimers();` but those messed up all the other test.